### PR TITLE
feat:update grub config to add hugepages

### PR DESCRIPTION
**Pull request by hemantsonawane95**

### CODE REVIEW

File: roles/grub/tasks/main.yml [hunk 1]
```diff
@@ -5,6 +5,13 @@
   when: cgroupsv2_disabled is defined and cgroupsv2_disabled is true
   tags: ['always']
 
+- name: Update GRUB configuration for hugepages
+  ansible.builtin.copy:
+    content: "GRUB_CMDLINE_LINUX_DEFAULT=\"{{ grub_config.cmdline_linux_default | join(' ') }}\""
```
Comment (freznicek) [inline roles/grub/tasks/main.yml:10]: `grub_config.cmdline_linux_default` is variable which may not be defined (above you define it as the group_vars vairable for subset of nodes), therefore I'd except `when` keyword in this task similar to the task above (line 5)

**Review by freznicek — CHANGES_REQUESTED** [review]
few details need to be tweaked, see details in code.

File: roles/grub/tasks/main.yml [hunk 2]
```diff
@@ -5,6 +5,13 @@
...
+    dest: /etc/default/grub.d/99-custom-settings.cfg
+  become: true
+  notify: update-grub
```
[the remark this answers is not in the document]
Comment (hemantsonawane95) [reply roles/grub/tasks/main.yml:14]: done

**Review by freznicek — APPROVED** [review]
LGTM

**Pull request merged**